app/views/csv_exports/new_users.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Request a CSV Report</h1>
+
+<section>
+  <div class="csv-users-filters clearfix">
+    <%= form_tag users_csv_exports_path(format: :csv), method: :get, class: "form-horizontal" do %>
+      <div><strong>User Permission Reports</strong></div>
+      <div class="form-group">
+        <%= label_tag :inherited, "Show inherited project permissions", class: "col-lg-2 control-label" %>
+        <div class="col-lg-2">
+          <%= select_tag :inherited, options_for_select([['No', false], ['Yes', true]], false), class: "form-control" %>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= label_tag :deleted, "Include deleted users", class: "col-lg-2 control-label" %>
+        <div class="col-lg-2">
+          <%= select_tag :deleted, options_for_select([['No', false], ['Yes', true]], false), class: "form-control" %>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= label_tag :project_id, "Project", class: "col-lg-2 control-label" %>
+        <% projects = [['', nil]] + Project.pluck(:name, :id) %>
```

Comment:
Kept it id to be consistent with the deploys report, since that reports the permalink.  

If we really should be using permalink instead, I can update both reports in a separate PR.